README.md

```diff
@@ -25,10 +26,26 @@ Guide.
 
 ## Official releases
 
-Release versions of Spine Toolbox can be found 
+### With Python installed
+
+Spine Toolbox is available on [The Python Package Index (PyPI)](https://pypi.org/project/spinetoolbox/).
+With Python installed in your system, you can get the latest release using [pipx](https://pypa.github.io/pipx/).
+
```

Comment:
How about adding two small steps here to first install pipx and then update PATH. 

```
pip install pipx
pipx ensurepath
```

Reply:
done